tests/std/tests/P0896R4_ranges_alg_minmax/test.cpp

```diff
@@ -334,6 +336,22 @@ constexpr void mm_constexpr_tests() {
         ProxyRef::no>>();
 }
 
+void test_for_issue1893() {
+    string val           = "meow";
+    string low           = "m";
+    string high          = "n";
+    int projection_count = 0;
+    const auto clamped   = ranges::clamp(
+        ref(val), ref(low), ref(high), [](auto x, auto y) { return x < y; },
+        [&projection_count](const auto& x) -> decltype(auto) {
+            ++projection_count;
+            return x.get();
+        });
+    (void) clamped;
+    ASSERT(projection_count <= 5); // 3 + 2 (STL_ASSERT)

```

Comment:
I think we should make this a tight bound:
```suggestion
#ifdef _DEBUG
    ASSERT(projection_count == 5);
#else
    ASSERT(projection_count == 3);
#endif
```